content/en/docs/automatic/_index.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Automatic & Assisted Instrumentation
+aliases: [/docs/zero-code]
+redirects:
+  - { from: /docs/instrumentation/*, to: ':splat' } # Only for `en`
+  - { from: 'net/*', to: 'dotnet/:splat' }
+weight: 260
+---
+
+Besides **code-based** instrumentation using the
+[language-specific APIs and SDKs](/docs/languages/) the OpenTelemetry project
+provides a set of solutions for **automatic** (or **zero-code**) and
+**assisted** instrumentation.
+
+You can learn about the difference between the three solutions and about their
```

Comment:
Won't hurt to restate the solutions here.

Reply:
```suggestion
You can learn about the difference between code-based, automatic and assisted instrumentation, and about their
```

like that, is the second comma put correctly?

Reply:
Yeah!